Source: written by Pierre Lejeune <superheron@gmail.com>
Upstream: submitted, see issue 7
Reason: fix compilation issues with xerces-c-3
--- beid-middleware-3.5.2-source-5800.orig/_src/eidmw/eidgui/mainwnd.h	2010-10-21 20:57:57.507375933 +0200
+++ beid-middleware-3.5.2-source-5800/_src/eidmw/eidgui/mainwnd.h	2010-12-21 16:06:15.966526679 +0100
@@ -53,7 +53,7 @@
 {
 public:
 	CallBackData( void )
-		: m_readerName(NULL)
+		: m_readerName((const char*)NULL)
 		, m_mainWnd(NULL)
 		, m_cardID(0)
 	{
--- beid-middleware-3.5.2-source-5800.orig/_src/eidmw/cardlayer/PCSC.h	2010-10-21 20:57:52.414376114 +0200
+++ beid-middleware-3.5.2-source-5800/_src/eidmw/cardlayer/PCSC.h	2010-12-21 17:43:50.854195500 +0100
@@ -40,6 +40,13 @@
 
 #include "wintypes.h"
 #include "Reader.h"
+
+// fix for gc issue #47 - compatible with pcsc-lite-1.6.4
+//  but still works on older pcsc-lite versions which *do* define SCARD_READERSTATE_A themselves
+#ifndef SCARD_READERSTATE_A
+#define SCARD_READERSTATE_A SCARD_READERSTATE
+#endif
+
 #ifndef SCARD_READERSTATEA
 #define SCARD_READERSTATEA SCARD_READERSTATE_A
 #endif

